src/helpers.ts

```diff
@@ -152,46 +175,84 @@ function noOp(_max: number, value: number) {
 }
 
 /**
- * Compute the crop and zoom from the croppedAreaPixels
+ * Compute crop and zoom from the croppedAreaPercentages.
+ */
+export function getInitialCropFromCroppedAreaPercentages(
+  croppedAreaPercentages: Area,
```

Comment:
I think that having a single object argument would help to understand what each parameter is when using this function.

Reply:
Do you mean to use only one object as a parameter and pass values through this object? I'm not opposed to the idea, but then we need to refactor all the functions in this fashion? I just reused the same coding style the lib uses, for example in `getInitialCropFromCroppedAreaPixels`

Reply:
Yes this is what I meant. You're right, it would be better to refactor everything at once. We can keep this for later 🙂